lib/avro/builder/types/enum_type.rb

```diff
@@ -27,12 +27,19 @@ def to_h(reference_state)
         def validate!
           super
           validate_required_attribute!(:symbols)
+          validate_enum_default!
         end
 
         private
 
+        def validate_enum_default!
```

Comment:
What happens if a client tries to set the `default` option when using Avro 1.9?

Reply:
I made a small update to the README.

If this gem is used with Avro 1.9 then it will still allow schemas to be defined with enum defaults. If the resulting schemas are used with a version of Ruby Avro prior to v1.10 then the default attribute for the enum will be ignored.

The Avro schema parser is permissive when it comes to additional attributes in the JSON. This is similar to the situation that we've been in for a while with aliases. They have been supported by this gem, but support for aliases was also added to Ruby Avro in v1.10.

Reply:
@tjwp looks like your README changes haven't been pushed yet

Reply:
Just pushed